frontend/src/components/RequestTargetInputs.vue

```diff
@@ -93,6 +116,44 @@ const inputValues: WritableComputedRef<RequestTarget> = computed({
   },
 });
 
+const computedRules = computed(() => {
+  const toAddressRules = {
+    $autoDirty: true,
+    required: helpers.withMessage('Destination address is required', required),
```

Comment:
I would argue that the placeholders value is there with the goal to provide some meaningful info to the user but not necessarily to the developer. 
Buut, looking at this from another POV, since there is a `selectedTargetAddress` property inside the `inputValues` i guess renaming it to `targetAddressRules` so it matches that property's name makes sense.

Reply:
Hm I think I wasn't precise enough here. I meant the helper message as this one is shown to the user and mismatches the placeholder.

But the renaming of the variable is also good. I think in general we moved more from the "to" towards the "target" prefix approach.